app/src/main/java/io/github/trojan_gfw/igniter/servers/fragment/ServerListAdapter.java

```diff
@@ -95,7 +95,7 @@ public void onClick(View v) {
 
     public void bindData(TrojanConfig config) {
         this.mConfig = config;
-        mRemoteAddrTv.setText(config.getRemoteAddr());
+        mRemoteAddrTv.setText(config.getRemoteServerRemark());
```

Comment:
How about renaming mRemoteAddrTv to something like mRemoteServerRemarkTv

Reply:
Yes, you are right.

Reply:
Done